Scribe/AboutTab/AboutTableData.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AboutTableData.swift
+//  Scribe
+//
+//  Created by Saurabh Jamadagni on 05/06/23.
+//
+
+import Foundation
+
+struct AboutTableData {
+  static var aboutTableData = [
+    ParentTableCellModel(
+      headingTitle: "Community",
+      section: [
+        Section(sectionTitle: "See the code on GitHub", imageString: "github", hasToggle: false),
+        Section(sectionTitle: "Chat with the team on Matrix", imageString: "matrix", hasToggle: false),
+        Section(sectionTitle: "Wikimedia and Scribe", imageString: "wikimedia", hasToggle: false),
+        Section(sectionTitle: "Share Scribe", imageString: "square.and.arrow.up", hasToggle: false)
```

Comment:
Hey hey :wave: 

Would texts like these, the `sectionTitle`s here, be good candidates for i18n? I see some support for it started already under [`./Scribe/AppTexts`](https://github.com/scribe-org/Scribe-iOS/tree/main/Scribe/AppTexts). Thinking that the app texts here could be good ways as well to continue working i18n into Scribe.

---
Implementing via the below perhaps?
- https://developer.apple.com/documentation/swiftui/preparing-views-for-localization
- https://developer.apple.com/documentation/swiftui/localizedstringkey